src/iam.ts

```diff
@@ -1415,6 +1415,45 @@ export class IAM extends IAMBase {
     ));
   }
 
+  async createOnChainProof(role: string, version: number, expiry: number, subject: string): Promise<string> {
```

Comment:
If adding a public method, please add documentation for it

Reply:
Good you noticed. This member should be private